.github/scripts/deploy-userguide

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+## Deploy codeigniter4/userguide
+
+# Setup variables
+SOURCE=$1
+TARGET=$2
+RELEASE=$3
+VERSION=`echo "$RELEASE" | cut -c 2-`
+
+echo "Preparing for version $3"
+echo "Merging files from $1 to $2"
+
+# Prepare the source
+cd $SOURCE
+git checkout master
+cd user_guide_src
+make html
+make epub
+
+# Prepare the target
+cd $TARGET
+git checkout master
+rm -rf docs
+
+# Copy files
+cp -Rf ${SOURCE}/user_guide_src/build/html ./docs
+cp -Rf ${SOURCE}/user_guide_src/build/epub/CodeIgniter.epub ./CodeIgniter${VERSION}.epub
+
+# Ensure underscore prefixed files are published
+touch ${TARGET}/docs/.nojekyll
```

Comment:
The error was reported:
> touch: cannot touch '/home/runner/work/CodeIgniter4/CodeIgniter4/source/docs/.nojekyll': No such file or directory

Reply:
Yes that was with the previous version of the script - I had incorrectly pasted `${SOURCE}` here.

Reply:
I got it.

Reply:
Updated run with the latest version of the script: https://github.com/MGatner/CodeIgniter4/runs/5463953690?check_suite_focus=true

Reply:
I've confirmed it. 👍